openpype/hosts/maya/plugins/load/load_gpucache.py

```diff
@@ -11,7 +11,7 @@ class GpuCacheLoader(load.LoaderPlugin):
     """Load Alembic as gpuCache"""
 
     families = ["model", "animation", "proxyAbc", "pointcache"]
-    representations = ["abc"]
+    representations = ["gpu_cache"]
```

Comment:
We've actually in the past used regular alembic representations in gpuCaches. Actually we still do when e.g. loading an Alembic with certain types of data from Houdini when rendering with Arnold.

This would break that logic for us. I understand how having both `abc` and `gpu_cache` might look polluted/clumsy in the Loader so I understand the decision but we'd still need to be able to load regular alembic files too - for us it's actually more important than to be able to load a slightly optimized playback format that is an exported maya gpuCache (which is a maya output format only).